src/components/profile/UserDetailForm/countries.json

```diff
@@ -0,0 +1,285 @@
+  { "id": 688, "name": "Serbia", "alpha2": "rs", "alpha3": "srb" },
+  { "id": 690, "name": "Seychelles", "alpha2": "sc", "alpha3": "syc" },
+  { "id": 694, "name": "Sierra Leone", "alpha2": "sl", "alpha3": "sle" },
+  { "id": 702, "name": "Singapore", "alpha2": "sg", "alpha3": "sgp" },
+  { "id": 703, "name": "Slovakia", "alpha2": "sk", "alpha3": "svk" },
+  { "id": 705, "name": "Slovenia", "alpha2": "si", "alpha3": "svn" },
+  { "id": 90, "name": "Solomon Islands", "alpha2": "sb", "alpha3": "slb" },
+  { "id": 706, "name": "Somalia", "alpha2": "so", "alpha3": "som" },
+  { "id": 710, "name": "South Africa", "alpha2": "za", "alpha3": "zaf" },
+  { "id": 728, "name": "South Sudan", "alpha2": "ss", "alpha3": "ssd" },
+  { "id": 724, "name": "Spain", "alpha2": "es", "alpha3": "esp" },
+  { "id": 144, "name": "Sri Lanka", "alpha2": "lk", "alpha3": "lka" },
+  { "id": 729, "name": "Sudan", "alpha2": "sd", "alpha3": "sdn" },
+  { "id": 740, "name": "Suriname", "alpha2": "sr", "alpha3": "sur" },
+  { "id": 752, "name": "Sweden", "alpha2": "se", "alpha3": "swe" },
+  { "id": 756, "name": "Switzerland", "alpha2": "ch", "alpha3": "che" },
+  {
+    "id": 760,
+    "name": "Syrian Arab Republic",
+    "alpha2": "sy",
+    "alpha3": "syr"
+  },
+  { "id": 762, "name": "Tajikistan", "alpha2": "tj", "alpha3": "tjk" },
+  {
+    "id": 834,
+    "name": "Tanzania, United Republic of",
+    "alpha2": "tz",
+    "alpha3": "tza"
+  },
+  { "id": 764, "name": "Thailand", "alpha2": "th", "alpha3": "tha" },
+  { "id": 626, "name": "Timor-Leste", "alpha2": "tl", "alpha3": "tls" },
+  { "id": 768, "name": "Togo", "alpha2": "tg", "alpha3": "tgo" },
+  { "id": 776, "name": "Tonga", "alpha2": "to", "alpha3": "ton" },
+  { "id": 780, "name": "Trinidad and Tobago", "alpha2": "tt", "alpha3": "tto" },
+  { "id": 788, "name": "Tunisia", "alpha2": "tn", "alpha3": "tun" },
+  { "id": 792, "name": "Turkey", "alpha2": "tr", "alpha3": "tur" },
+  { "id": 795, "name": "Turkmenistan", "alpha2": "tm", "alpha3": "tkm" },
+  { "id": 798, "name": "Tuvalu", "alpha2": "tv", "alpha3": "tuv" },
+  { "id": 800, "name": "Uganda", "alpha2": "ug", "alpha3": "uga" },
+  { "id": 804, "name": "Ukraine", "alpha2": "ua", "alpha3": "ukr" },
+  {
+    "id": 784,
+    "name": "United Arab Emirates",
+    "alpha2": "ae",
+    "alpha3": "are"
+  },
+  {
+    "id": 826,
+    "name": "United Kingdom of Great Britain and Northern Ireland",
+    "alpha2": "gb",
+    "alpha3": "gbr"
+  },
+  {
+    "id": 840,
+    "name": "United States of America",
+    "alpha2": "us",
+    "alpha3": "usa"
+  },
+  { "id": 858, "name": "Uruguay", "alpha2": "uy", "alpha3": "ury" },
+  { "id": 860, "name": "Uzbekistan", "alpha2": "uz", "alpha3": "uzb" },
+  { "id": 548, "name": "Vanuatu", "alpha2": "vu", "alpha3": "vut" },
+  {
+    "id": 862,
+    "name": "Venezuela (Bolivarian Republic of)",
+    "alpha2": "ve",
+    "alpha3": "ven"
+  },
+  { "id": 704, "name": "Viet Nam", "alpha2": "vn", "alpha3": "vnm" },
```

Comment:
this is a typo

Reply:
I did check all the countries, shall I check it?